lib/srv/db/common/auth.go

```diff
@@ -495,7 +495,9 @@ func (a *dbAuth) GetTLSConfig(ctx context.Context, sessionCtx *Session) (*tls.Co
 // getTLSConfigVerifyFull returns tls.Config with full verification enabled ('verify-full' mode).
 // Config also includes database specific adjustment.
 func (a *dbAuth) getTLSConfigVerifyFull(ctx context.Context, sessionCtx *Session) (*tls.Config, error) {
-	tlsConfig := &tls.Config{}
+	tlsConfig := &tls.Config{
+		MinVersion: tls.VersionTLS12,
+	}
```

Comment:
`getTLSConfigVerifyFull` is used for Teleport Database Service to connect to actual database servers. I think this can potentially break if the database server cannot do TLS 1.2. Hopefully no such database servers exist in the field. We might want to update the documentation though as TLS 1.2 will be the minimum requirement or maybe a breaking change for v14.